source/draw/RasterRgba.ooc

```diff
@@ -131,8 +125,6 @@ RasterRgba: class extends RasterPacked {
 		result swapRedBlue()
 		result
 	}
-	_createCanvas: override func -> Canvas { RasterRgbaCanvas new(this) }
-
 	operator [] (x, y: Int) -> ColorRgba { this isValidIn(x, y) ? ((this buffer pointer + y * this stride) as ColorRgba* + x)@ : ColorRgba new(0, 0, 0, 0) }
```

Comment:
Not for this PR, but should `[]` really always call `isValidIn`? Seems pretty inefficient.